lib/bundles/inspec-compliance/api.rb

```diff
@@ -43,14 +56,19 @@ def self.profiles(config)
     # return the server api version
     # NB this method does not use Compliance::Configuration to allow for using
     # it before we know the version (e.g. oidc or not)
-    def self.version(url, insecure)
+    def self.version(config)
+      url = config['server']
+      insecure = config['insecure']
+
       if url.nil?
         puts "
 Server configuration information is missing.
 Please login using `inspec compliance login https://compliance.test --user admin --insecure --token 'PASTE TOKEN HERE' `
 "
```

Comment:
API implementation may throw an error but will never exit, this can only be done by CLI